src/thor/bidirectional_astar.cc

```diff
@@ -1297,8 +1297,9 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
     // bidirectional a* has a bug where it fails trivial routes in which you are on a one way edge and
     // the origin is near the end of the edge and the destination is near the beginning, in other
     // words a route that looks trivial but actually needs to go around the block to complete
-    if (path_edges.size() == 1)
+    if (path_edges.size() == 1) {
       LOG_WARN("Trivial route with bidirectional A* should not be allowed");
+    }
```

Comment:
newer gcc versions also have a warning for this. I really don't like if/for without braces for some reason, but never thought about this situation, which is a very good reason:) probably should turn that warning on at some point

Reply:
hahah :+1: